src/hooks/usePrevious.tsx

```diff
@@ -0,0 +1,11 @@
+import { useEffect, useRef } from 'react'
+
+export function usePrevious<T>(value: T): T | undefined {
+	const ref = useRef<T>()
+
+	useEffect(() => {
+		ref.current = value
+	}, [value])
+
+	return ref.current
+}
```

Comment:
What about using [@react-hook/previous](https://www.npmjs.com/package/@react-hook/previous)?

Reply:
OK, I can use that 👍  I didn't know the package. The implementation is basically the same. I just remembered I used it like that once in the past without any package.